tldr/analysis.py

```
@@ -100,10 +100,21 @@ def impact_analysis(
 
     # Find target function(s) as callees (functions being called)
+    # For PHP, names may be ClassName::method — match both qualified and bare names
+    def _matches_target(func_name: str, target: str) -> bool:
+        if func_name == target:
+            return True
+        # Match bare name against qualified ClassName::method
+        if "::" in func_name and func_name.split("::")[-1] == target:
+            return True
+        if "::" in target and target.split("::")[-1] == func_name:
+            return True
+        return False
```

Ruff (0.15.7)

[warning] Return the condition directly

Inline condition

(SIM103)